### ISSUE TITLE
New bug with equation rendering

### ISSUE THREAD

**Issue by sethaxen** [tlienart/Franklin.jl#726]
v0.10.6 introduced a bug in the TeX parsing or rendering. Here's a minimal example:
```julia
julia> using Franklin

julia> s = raw"""
           \newcommand{\dtan}[1]{\dot{#1}}
           $$\dtan{x}$$
           """ |> fd2html # works on both versions
"\\[\\dot{ x}\\]\n"

julia> s = raw"""
           \newcommand{\dtan}[1]{\dot{#1}}
           $$\frac{\dtan{x}}{x}$$
           """ |> fd2html
```

on v0.10.5:
```julia
"\\[\\frac{\\dot{ x}}{x}\\]\n"
```

on v0.10.6 and later:
```julia
ERROR: Franklin.LxObjError("Command/Environment '\\dtan' expects 1 argument(s) and there\nshould be no space(s) between the command name and the first brace:\n\\com{arg1}... or \\begin{env}{arg1}...\n")
Stacktrace:
 [1] find_opts_braces(::Franklin.Token, ::Int64, ::Array{Franklin.OCBlock,1}, ::SubString{String}) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/parser/latex/blocks.jl:197
 [2] find_lxcoms(::Array{Franklin.Token,1}, ::Array{Franklin.LxDef,1}, ::Array{Franklin.OCBlock,1}, ::Int64; inmath::Bool) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/parser/latex/blocks.jl:265
 [3] convert_md_math(::SubString{String}, ::Array{Franklin.LxDef,1}, ::Int64) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/markdown/md.jl:227
 [4] convert_math_block(::Franklin.OCBlock, ::Array{Franklin.LxDef,1}) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/markdown/blocks.jl:102
 [5] convert_block(::Franklin.OCBlock, ::Array{Franklin.LxDef,1}) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/markdown/blocks.jl:30
 [6] convert_inter_html(::String, ::Array{Franklin.AbstractBlock,1}, ::Array{Franklin.LxDef,1}) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/markdown/md.jl:396
 [7] convert_md(::String, ::Array{Franklin.LxDef,1}; isrecursive::Bool, isinternal::Bool, isconfig::Bool, has_mddefs::Bool, pagevar::Bool, nostripp::Bool) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/markdown/md.jl:167
 [8] fd2html_v(::String; internal::Bool, dir::String, nop::Bool) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/html/html.jl:62
 [9] fd2html_v at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/html/html.jl:50 [inlined]
 [10] fd2html(::String; kw::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/html/html.jl:89
 [11] fd2html at /Users/saxen/.julia/packages/Franklin/poSdQ/src/converter/html/html.jl:89 [inlined]
 [12] |>(::String, ::typeof(fd2html)) at ./operators.jl:834
 [13] top-level scope at REPL[6]:1
```

**Comment by tlienart** [comment]
hmm that's unfortunate, thanks again for the report, I'll investigate

**Comment by tlienart** [comment]
closed by #727
